library/std/src/sys_common/wtf8.rs

```diff
@@ -935,7 +935,7 @@ impl<'a> Iterator for Wtf8CodePoints<'a> {
 
 /// Generates a wide character sequence for potentially ill-formed UTF-16.
 #[stable(feature = "rust1", since = "1.0.0")]
-#[derive(Clone)]
+#[derive(Clone, Debug)]
```

Comment:
This addition is an insta-stable API change.

r? libs-api

Reply:
This one is outdated, but there are others.

Reply:
New impls are fine on stuff that's not `pub` and `stable`.

(We have a lot of `pub` stuff that really ought to be `pub(crate)`, but they're not publicly reachable or they'd be flagged for missing stability attributes.)

Reply:
Does that mean this no longer needs libs-api review?